### PR TITLE
Vue 1.0 fixes

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR apply some fixes related to vue.js 1.0 migration.

### CODE REVIEW

File: js/components/form/base-completer.vue [hunk 1]
```diff
@@ -99,10 +110,7 @@ export default {
         }
     },
     beforeDestroy: function() {
```
Comment (vinyll) [inline js/components/form/base-completer.vue]: `beforeDestroy() {`

File: js/mixins/modal.js [hunk 2]
```diff
@@ -1,29 +1,22 @@
...
-        methods: {
-            hide: function() {
-                $el(this).modal('hide');
-            }
-        }
-    }
-
-});
+export default {
+     ready: function() {
```
Comment (vinyll) [inline js/mixins/modal.js]: `ready() {`

File: js/mixins/modal.js [hunk 3]
```diff
@@ -1,29 +1,22 @@
...
+             .on('hide.bs.modal', (e) => {
+                 this.$dispatch('modal:close');
+             })
+             .on('hidden.bs.modal', () => {
+                 this.$dispatch('modal:closed');
+                 this.$destroy(true);
+             });
+     },
+     methods: {
+         close: function() {
```
Comment (vinyll) [inline js/mixins/modal.js]: `close() {`

**Comment by davidbgk** [comment]
r+

**Comment by davidbgk** [comment]
Community resources submissions still doesn't end up as expected.

**Comment by noirbizarre** [comment]
Yep, I will submit another PR because this is not vue-related but backend-related. It need fixes on API and a data migration.

**Comment by davidbgk** [comment]
Alright, r+

**Comment by vinyll** [comment]
:clap:

**Pull request merged**